python/Dockerfile

```diff
@@ -36,39 +38,96 @@ RUN apt-get update \
 
 COPY --chown=dependabot:dependabot python/helpers /opt/python/helpers
 USER root
+# TODO: Now that switched from `pyenv install` which compiled from source to downloading / copying a pre-compiled python
+# we could entirely drop pyenv if we change our ruby code that calls `pyenv exec` to track which version of python to
+# call and uses the full python paths.
 ENV PYENV_ROOT=/usr/local/.pyenv \
   PATH="/usr/local/.pyenv/bin:$PATH"
```

Comment:
I think you can remove this line that updates the PATH env

Reply:
Don't we need this so the `pyenv` shims take precedence on the `PATH`?

https://github.com/pyenv/pyenv#set-up-your-shell-environment-for-pyenv

Reply:
Yep you're right we do. I thought this was adding the actual python installations to the PATH